### PR TITLE
Documentation and metadata

### PR DESCRIPTION
**Pull request by Elisa-on-GitHub**
This PR adds the topics Data Documentation and Metadata as separate topics. There was an earlier PR that proposed to do this, #110 but that was closed when we realised we needed more time to implement changes and introduce these topics as two separate topics. The current topics were updated using both the input of the comments in #110 and from the metadata specialist.

Is there a way to give the metadata specialist credit as a contributor? I don't know his github handle and he's an inactive github user.

### CODE REVIEW

**Comment by chartgerink** [comment]
Thanks @Elisa-on-GitHub 🙌  You can add the name directly to the `about.qmd` file, if you'd like. There is a section for those contributing outside of GitHub 😊

It seems like the branch got outdated, so I will handle the conflicts for you on this PR.

**Review by Karvovskaya — APPROVED** [review]
Great to see such an improvement of the documentation part!

File: topics/metadata.qmd [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+Controlled vocabularies are lists of terms created by domain experts to refer to a specific phenomenon or event. Controlled vocabularies are intended to reduce ambiguity that is inherent in normal human languages where the same concept can be given different names and to ensure consistency. Controlled vocabularies are used in subject indexing schemes, subject headings, thesauri, taxonomies and other knowledge organisation systems. Some vocabularies are very internationally accepted and standardized and may even become an ISO standard or a regional standard/classification. Controlled vocabularies can be broad in scope or very limited to a specific field. When a Data Management Plan template includes a question on the used ontology (if any), what is usually meant is: is there a specific vocabulary or classification system used. The National Bioinformatics Infrastructure Sweden gives some more explanation about controlled vocabularies and ontologies [here](https://nbisweden.github.io/module-metadata-dm-practices/02-ontologies/index.html). In short, an ontology does not only describe terms, but also indicates relationships between these terms.
```
Comment (Jolien-S) [inline topics/metadata.qmd:20]:
```suggestion
Controlled vocabularies are lists of terms created by domain experts to refer to a specific phenomenon or event. Controlled vocabularies are intended to reduce ambiguity that is inherent in normal human languages where the same concept can be given different names and to ensure consistency. Controlled vocabularies are used in subject indexing schemes, subject headings, thesauri, taxonomies and other knowledge organisation systems. Some vocabularies are very internationally accepted and standardised and may even become an ISO standard or a regional standard/classification. Controlled vocabularies can be broad in scope or very limited to a specific field. When a Data Management Plan template includes a question on the used ontology (if any), what is usually meant is: is there a specific vocabulary or classification system used? The National Bioinformatics Infrastructure Sweden gives some more explanation about controlled vocabularies and ontologies [here](https://nbisweden.github.io/module-metadata-dm-practices/02-ontologies/index.html). In short, an ontology does not only describe terms, but also indicates relationships between these terms.
```

**Review by Jolien-S — CHANGES_REQUESTED** [review]
Great improvement of information about metadata and documentation! I just made two spelling suggestions; after they have been implemented, we can approve this PR.

**Comment by chartgerink** [comment]
Thanks for the ping @Jolien-S 😊 I started working on this in the afternoon, but the conflict is substantial. Do you have any guidance on how best to decide what has priority? 

The structure doesn't necessarily easily fit, so there are some judgment calls. Should i try to merge them as much possible, or should i let the new version take priority and flesh it out with anything that might've gotten lost in the meantime (like links)?

With your guidance I can move forward 🙏

**Review by chartgerink — APPROVED** [review]

**Pull request merged**